dsp/window/window.go

```diff
@@ -0,0 +1,286 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package window
+
+import (
+	"math"
+)
+
+// Rectangle - modifies the seq in place by Rectangle window and returns the seq.
+//
+// Rectangle window is a base high-resolution window,
+// which result is correspond to a selection of limited length sequence of values without any modification
+// (so, it doesn't modifies the seq at all).
+// Rectangle window has the lowest width of the main lobe and largest level of the side lobes.
+//
+// Indicators of quality:
+//  ΔF_0	=  2
+//  ΔF_0.5	=  0.89
+//  K	=  1
+//  ɣ_max	= -13
+//  β	=  0
+func Rectangle(seq []float64) []float64 {
+	return seq
+}
+
```

Comment:
For the non-trivial window functions, can you add links to references? Also the formula for each.